server.js

```diff
@@ -0,0 +1,41 @@
+const http = require('http');
+const fs = require('fs');
+const PORT = process.env.PORT || 8080;
+
+
+const server = http.createServer((request, response) => {
+  let method = request.method;
+  let url = request.url;
+
+  if (url.indexOf('public') < 1) { //Add public to url
+    url = 'public' + request.url;
+  }
+
+  console.log(url);
+  console.log('Method: ', method);
+
+  switch (url) {
+    case 'public/':
+      fs.readFile('public/index.html', (err, data) => {
+        let data2 = data.toString();
+        response.writeHead(200, { 'Content-Type': 'text/html' })
+        response.write(data2)
+        response.end();
+      });
+      break;
+    case `public${request.url}`:
+      fs.readFile(url, (err, data) => {
```

Comment:
Your can't hardcode your content type or your status at this point. How do you respond to a CSS file request if the type is set to html. Also, you will not always have the file someone requests. You will need to return 404 if someone asks for `/kfjdslkjfs.html` so you can not hard code 200.